### PR TITLE
Add Linux udev rule for Input club keyboards to build FAQ

### PR DESCRIPTION
**Pull request by r2d2rogers**
Is this a good place to put a keyboard specific rule or would it be better in the readme for the keyboard or sub project?

### CODE REVIEW

**Comment by r2d2rogers** [comment]
I've added similar changes to the infinity section of the ergodox readme, for comparison.

**Comment by fredizzimo** [comment]
And this PID/VID, applies to the bootloader, which in this case is the kiibohd bootloader. So it applies to at least the Ergodox Infinity, and Infinity 60. The Whitefox is probably also using the same.

**Comment by r2d2rogers** [comment]
I saw that it is possible to configure the PID/VID in a config.h file.

When the make target for flashing a keyboard is called can it find the PID/VID that will be compiled and compare with the values in the udev rules on the system and output the required udev rules either as a message on screen or in a file if they are not setup in /etc?

The udev directory is readable by a regular user.

**Comment by fredizzimo** [comment]
The PID/VID that you configure in `config.h` is not the same as the one used for flashing. 

The `config.h` version is the one that QMK reports, while the bootloader one is used for flashing. 

The good news is that we don't have too many different bootloaders to support. 
* The Teensy ones (but they don't need a special driver)
* The LUFA
* The AVR standard ones
* Kiibohd for the Infinity Keyboards
* Maybe some other ARM ones.

The list of PIDs that you need to add to the rules, should pretty much match the list in [drivers.txt](https://github.com/qmk/qmk_firmware/blob/master/util/drivers.txt). Although, I think that list is not totally complete, the LUFA bootloader for example is probably missing. @jackhumbert, can you check that, especially since you made an improved version of it. I don't know which bootloader the Whitefox uses, and the keyboard that @skullydazed is working on, probably also need its bootloader listed there.

**Comment by Talljoe** [comment]
Note that `1c11` seems to work for all Infinity bootloaders (i.e. my Infinity60) not just the Ergodox.

**Comment by fredizzimo** [comment]
@jackhumbert, I don't think so, these are rules that have to be added to Linux itself

@r2d2rogers, from my point this looks good, but as @Talljoe pointed out, we should refer to all InputClub keyboards

**Comment by drashna** [comment]
@r2d2rogers If you wanted to update this, and rebase it to fix the merge conflicts?

**Comment by drashna** [comment]
@r2d2rogers would you mind updating this? 

Specifically, the merge conflicts need to be addressed, and some of the files no longer exist.

File: docs/faq_build.md [hunk 1]
```diff
@@ -60,7 +60,12 @@ SUBSYSTEMS=="usb", ATTRS{idVendor}=="03eb", ATTRS{idProduct}=="2ff0", MODE:="066
 # tmk keyboard products     https://github.com/tmk/tmk_keyboard
 SUBSYSTEMS=="usb", ATTRS{idVendor}=="feed", MODE:="0666"
 ```
+Or for Infinity Ergodox:
```
Comment (drashna) [inline docs/faq_build.md]: This should be for all of the Input Club boards, not just the Ergodox Infinity.

File: keyboards/ergodox/readme.md [hunk 2]
```diff
@@ -101,6 +101,16 @@ To flash with ´teensy-loader-cli´:
 
 ## ErgoDox Infinity
 
+## Linux udev rules
```
Comment (drashna) [inline keyboards/ergodox/readme.md]: This file has moved.

File: docs/faq_build.md (hunk 1 above)
Comment (r2d2rogers) [reply docs/faq_build.md]: Adjusted labeling.

File: keyboards/ergodox/readme.md (hunk 2 above)
Comment (r2d2rogers) [reply keyboards/ergodox/readme.md]: Should I add the note to all the input club keyboard readme files?

**Comment by drashna** [comment]
Looks like the merge caused it to touch every file :(

**Comment by drashna** [comment]
@r2d2rogers Could you fix this PR, or close it and re-open a new one with just the desired change?

**Pull request closed without merging**